File: classes/parte-9-ingenieria-de-software-poliglota/147-integracion-continua-ci-multi-lenguaje/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	verde := true
	for _, s := range strings.Fields(line) {
		n, _ := strconv.Atoi(s)
		if n != 1 {
			verde = false
		}
	}
	res := "rojo"
	if verde {
		res = "verde"
	}
	fmt.Printf("ci=%s\n", res)
}

File: classes/parte-9-ingenieria-de-software-poliglota/147-integracion-continua-ci-multi-lenguaje/implementaciones/rust/main.rs
use std::io::Read;

fn main() {
    let mut s = String::new();
    std::io::stdin().read_to_string(&mut s).unwrap();
    let verde = s.split_whitespace().all(|x| x.parse::<i64>().unwrap() == 1);
    println!("ci={}", if verde { "verde" } else { "rojo" });
}
